src/pages/settings/AboutPage.js

```diff
@@ -29,6 +30,13 @@ const AboutPage = (props) => {
                 Navigation.navigate(ROUTES.SETTINGS_APP_DOWNLOAD_LINKS);
             },
         },
+        {
+            translationKey: 'initialSettingsPage.aboutPage.viewKeyboardShortcuts',
+            icon: Expensicons.Keyboard,
+            action: () => {
```

Comment:
Unnecessary arrow function

```suggestion
            action: KeyboardShortcuts.showKeyboardShortcutModal,
```